src/main/java/frc/robot/RobotContainer.java

```diff
@@ -50,6 +51,10 @@
     private Command sensitiveDriverControlCommand;
     private Logger logger = Logger.getLogger("RobotContainer");
 
+    private final double RIGHT_SPEED = 0.2;
```

Comment:
This should be in the constructor of the command, not here

Reply:
I put it in RobotContainer because they are values you are supposed to call on line 128 since I don't want to use "Magic Numbers". I'll add them there now.